src/components/footer.astro

```diff
@@ -67,21 +81,25 @@ import LinkedInIcon from '~icons/ph/linkedin-logo-fill';
 
     display: flex;
     justify-content: space-between;
-    align-items: baseline;
+    align-items: flex-start;
     flex-wrap: wrap;
   }
-
-  .content p:first-child {
-    font-size: 1.2rem;
+  .contact p:first-child {
+    margin-top: 1.2rem;
     max-width: 30ch;
   }
```

Comment:
This is to create a gap on the first child, right? couldn't that also be accomplished with a display:grid with grid-templates giving more space to the top one? :eyes:

Reply:
I think the design can be cleaned up, will be trying to clean it up.
This bit here seems to be something left over that doesn't affect the page right now.